openquake/hazardlib/gsim/sharma_2009.py

```diff
@@ -94,8 +94,6 @@ def get_mean_and_stddevs(self, sites, rup, dists, imt, stddev_types):
         for specification of input and result values.
         """
 
-        assert imt.__class__ in self.DEFINED_FOR_INTENSITY_MEASURE_TYPES
-
```

Comment:
Note that I've edited a file from another (merged) branch (Sharma) here to fix a redundancy @g-weatherill found in this branch.